config/gitter-config-schema.yaml

```diff
@@ -26,3 +26,8 @@ properties:
     type: boolean
   enable_metrics:
     type: boolean
+  config:
```

Comment:
"Config" is too generic a  name to refer to just logging?.

We don't seem to be specifying the "files" option that is used in the sample config  in this schema?